src/components/RwScrollToTop/RwScrollToTop.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { onBeforeMount, ref } from 'vue';
+import RwButton from '../RwButton/RwButton.vue';
+import RwIcon from '../RwIcon/RwIcon.vue';
```

Comment:
import components from components root '.../components'. It will reduce bundle size, because we bundle dynamic import to your component